46.md

```diff
@@ -0,0 +1,159 @@
+NIP-46
+======
+
+Nostr Connect
```

Comment:
Just a thought: maybe it makes sense to see this NIP more generic (or to extract a generic one out of this one) e.g. "Nostr RPC" - so that it is not just defining that "connect" mechanism, but a generic RPC mechanism. We just need to define here that this is always using JSON RPC. The "connect" mechanism could then be defined on top of this "Nostr RPC", by just defining the needed JSON RPC methods and its behaviour. For that "Nostr RPC" I think it could be very useful to define one method, that just returns all supported methods. Also other extensions/NIPs could probably be much easier specified, once we have such "Nostr RPC" NIP.